pkgs/development/python-modules/dominate/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "dominate";
+  version = "2.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0s9s9j9xmhkzw7apqx170fyvc0f800fd4a5jfn8xvj9k6vryd32b";
+  };
+
+  doCheck = !isPy3k;
```

Comment:
Changed this to reflect that tests only fail with python 3, gist of the error still here: https://gist.github.com/Infinisil/5d5d425d9b1d9eceed30ca4546382900#file-error-log-L46